catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/teams/TeamResourceTest.java

```diff
@@ -296,6 +324,16 @@ public void compareEntities(Team expected, Team updated, Map<String, String> aut
 
   @Override
   public void assertFieldChange(String fieldName, Object expected, Object actual) throws IOException {
-    assertCommonFieldChange(fieldName, expected, actual);
+    if (expected == actual) {
```

Comment:
Should this be `expected.equals(actual)` to compare the values of objects?

Reply:
This is actually == check. This catches both objects being null...